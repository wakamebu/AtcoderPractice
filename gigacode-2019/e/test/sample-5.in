1 100
5 60
50 7 90
