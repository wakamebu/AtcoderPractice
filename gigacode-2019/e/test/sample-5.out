17.142857142857142349612
